### PR TITLE
Remove Calligraphy

### PR DESCRIPTION
**Pull request by shadow7**
https://developer.android.com/guide/topics/ui/look-and-feel/downloadable-fonts.html

### CODE REVIEW

**Comment by f2rf2r** [comment]
Can we do some setup/config/example for the new font handling before removing Calligraphy?

File: app/src/main/res/layout/activity_example1.xml [hunk 1]
```diff
@@ -16,7 +16,6 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="Click"
-        android:textAppearance="@style/Roboto_Regular"
```
Comment (russhwolf) [inline app/src/main/res/layout/activity_example1.xml]: I think it'd be good to leave something with this functionality in place using the new font apis, for demo purposes.

**Comment by russhwolf** [comment]
I have a similar branch on the Java Skeleton now that does this while retaining the custom font usage via a font resource. Can port it here once it goes in there. 
https://github.com/IntrepidPursuits/skeleton-android/pull/51

**Comment by gdaniels** [comment]
What's up with this PR?

**Comment by russhwolf** [comment]
It can be closed. Was duplicated by #19 which is merged

**Pull request closed without merging**